R/write_point_dsg.R

```diff
@@ -2,8 +2,8 @@
 #'
 #'
 #'@param nc_file A string file path to the nc file to be created.
-#'@param times A vector of times. Must be of type \code{\link{POSIXct}} or an attempt to 
-#'convert it will be made using \code{\link{as.POSIXct(times)}}.
+#'@param times A vector of times. Must be of type \code{POSIXct} or an attempt to 
+#'convert it will be made using \code{as.POSIXct(times)}.
```

Comment:
There is a syntax to link to other pkgs functions if you want to use it. Involves [pkg]{function} I think